### PR TITLE
Add playbooks for updating IAM mappings

### PR DESCRIPTION
**Pull request by apdibbo**

### CODE REVIEW

File: AnsiblePlaybooks/ansible-openstack-tasks/files/template-mappings.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (anish-mudaraddi) [inline AnsiblePlaybooks/ansible-openstack-tasks/files/template-mappings.yml:1]: I'd recommend maybe putting these template files in their own directory - makes it more obvious what they are are less likely to be overwritten accidentally

File: AnsiblePlaybooks/ansible-openstack-tasks/files/prod_current_mappings.yml [hunk 2]
```diff
@@ -0,0 +1,154 @@
+current_mappings:
```
Comment (anish-mudaraddi) [inline AnsiblePlaybooks/ansible-openstack-tasks/files/prod_current_mappings.yml:1]: Should we maybe put these files in `.gitignore`? 
Reading your docs - you create these by running the `get-idp-mappings.yml` right?

Comment (anish-mudaraddi) [reply AnsiblePlaybooks/ansible-openstack-tasks/files/prod_current_mappings.yml:1]: This is the same for `staging_current_mappings.yml` and `new_mappings.yml` and `mappings.yml`

File: AnsiblePlaybooks/ansible-openstack-tasks/update-idp-mappings.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    - name: output age of mappings
+      debug:
+        msg: "{{ (ansible_date_time.epoch|float - mapping_stats.stat.mtime )  }}"
+    - name: remind user to fetch updated mappings
+      debug:
+        msg: "Mappings havent been fetched for a while, you should do that"
+      when: (ansible_date_time.epoch|float - mapping_stats.stat.mtime ) > 3600
+    - name: end if mappings are old
+      ansible.builtin.meta: end_play
+      when: (ansible_date_time.epoch|float - mapping_stats.stat.mtime ) > 3600
```
Comment (anish-mudaraddi) [inline AnsiblePlaybooks/ansible-openstack-tasks/update-idp-mappings.yml:18]: maybe its worth just fetching them by default and having a flag to ignore fetching when we run the playbook?

File: AnsiblePlaybooks/ansible-openstack-tasks/files/prod_current_mappings.yml (hunk 2 above)
Comment (apdibbo) [reply AnsiblePlaybooks/ansible-openstack-tasks/files/prod_current_mappings.yml]: I believe this should be resolved now

File: AnsiblePlaybooks/ansible-openstack-tasks/files/template-mappings.yml (hunk 1 above)
Comment (apdibbo) [reply AnsiblePlaybooks/ansible-openstack-tasks/files/template-mappings.yml]: I believe this should be resolved now

File: AnsiblePlaybooks/ansible-openstack-tasks/update-idp-mappings.yml (hunk 3 above)
Comment (apdibbo) [reply AnsiblePlaybooks/ansible-openstack-tasks/update-idp-mappings.yml:18]: Maybe but I think I'd like to keep it manual for now

**Review by anish-mudaraddi — APPROVED** [review]

**Pull request merged**